tests/test_memory.py

```diff
@@ -0,0 +1,30 @@
+import weakref
+
+from pympler.asizeof import asizeof as sizeof  # type: ignore
+from hypothesis import given
+
+from ppb_vector import Vector2
+from utils import floats, vectors
+
+
+class DummyVector:
+    """A naïve representation of vectors."""
+
+    x: float
+    y: float
+
+    def __init__(self, x, y):
+        self.x = float(x)
+        self.y = float(y)
+
+
+@given(x=floats(), y=floats())
+def test_object_size(x, y):
+    """Check that Vector2 is 3 times smaller than a naïve version."""
```

Comment:
Docstring doesn't match implementation

Reply:
Thanks.  I updated the test in the last commit (`Vector2` with a `__weakref__` slot was larger than 1/3rd of `DummyVector`) and forgot to update the commit.